app/controllers/admins/calculators_controller.rb

```diff
@@ -43,7 +39,7 @@ def update
     private
 
     def find_calculator
-      @calculator = Calculator.find(params[:id])
+      @calculator = Calculator.friendly.find(params[:id])
```

Comment:
why do we use `friendly` but parameter is called `id`?

Reply:
Because I used the same approach as was on `show` action